[bitnami/grafana-tempo] Update grafana-tempo-2 (#14917)
bitnami/charts: bitnami/grafana-tempo/README.md

## Upgrading

### To 2.0.0

> This major release, some configuration parameters (`search_enabled` and `metrics_generator_enabled`) have been removed and both are by default true.

Reviewer: ```suggestion
This major release removes `search_enabled` and `metrics_generator_enabled` configuration parameters, assuming they are always enabled.
```
